apps/src/gamelab/AnimationTab/PiskelEditor.jsx

```diff
@@ -132,6 +133,9 @@ const PiskelEditor = React.createClass({
   onPiskelReady() {
     this.isPiskelReady_ = true;
     this.loadSelectedAnimation_(this.props);
+    if (this.props.allAnimationsSingleFrame) {
+      this.piskel.toggleFrameColumn(true);
+    }
```

Comment:
Nit: `loadSelectedAnimation_` can be a long async process, it might be more intuitive if it's the last step in `onPiskelReady`.